mmv1/third_party/terraform/resources/resource_apigee_keystores_aliases_key_cert_file.go

```diff
@@ -0,0 +1,674 @@
+					Schema: map[string]*schema.Schema{
+						"cert_info": {
+							Type:        schema.TypeList,
+							Optional:    true,
+							Computed:    true,
+							Description: `List of all properties in the object.`,
+							Elem: &schema.Resource{
+								Schema: map[string]*schema.Schema{
+									"basic_constraints": {
+										Type:        schema.TypeString,
+										Optional:    true,
+										Computed:    true,
+										Description: `X.509 basic constraints extension.`,
+									},
+									"expiry_date": {
+										Type:        schema.TypeString,
+										Optional:    true,
+										Computed:    true,
+										Description: `X.509 notAfter validity period in milliseconds since epoch.`,
+									},
+									"is_valid": {
+										Type:     schema.TypeString,
+										Optional: true,
+										Computed: true,
+										Description: `Flag that specifies whether the certificate is valid. 
+Flag is set to Yes if the certificate is valid, No if expired, or Not yet if not yet valid.`,
+									},
+									"issuer": {
+										Type:        schema.TypeString,
+										Optional:    true,
+										Computed:    true,
+										Description: `X.509 issuer.`,
+									},
+									"public_key": {
+										Type:        schema.TypeString,
+										Optional:    true,
+										Computed:    true,
+										Description: `Public key component of the X.509 subject public key info.`,
+									},
+									"serial_number": {
+										Type:        schema.TypeString,
+										Optional:    true,
+										Computed:    true,
+										Description: `X.509 serial number.`,
+									},
+									"sig_alg_name": {
+										Type:        schema.TypeString,
+										Optional:    true,
+										Computed:    true,
+										Description: `X.509 signatureAlgorithm.`,
+									},
+									"subject": {
+										Type:        schema.TypeString,
+										Optional:    true,
+										Computed:    true,
+										Description: `X.509 subject.`,
+									},
+									"subject_alternative_names": {
+										Type:        schema.TypeList,
+										Optional:    true,
+										Computed:    true,
+										Description: `X.509 subject alternative names (SANs) extension.`,
+										Elem: &schema.Schema{
+											Type: schema.TypeString,
+										},
+									},
+									"valid_from": {
+										Type:        schema.TypeString,
+										Optional:    true,
+										Computed:    true,
+										Description: `X.509 notBefore validity period in milliseconds since epoch.`,
+									},
+									"version": {
+										Type:        schema.TypeInt,
+										Optional:    true,
+										Computed:    true,
+										Description: `X.509 version.`,
+									},
+								},
+							},
+						},
+					},
+				},
+			},
+			"key": {
+				Type:        schema.TypeString,
+				ForceNew:    true,
+				Optional:    true,
+				Description: `Private Key content, omit if uploading to truststore`,
+			},
+			"password": {
```

Comment:
If there's a password field, I think it's necessary to make this field sensitive. `Sensitive: true`